dev-libs/spsdeclib/spsdeclib-5.1.ebuild

```diff
@@ -0,0 +1,44 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit autotools eutils unpacker
+
+DESCRIPTION="Kryoflux SPS Decoder Library"
+HOMEPAGE="https://www.kryoflux.com/"
+SRC_URI="https://www.kryoflux.com/download/${PN}_${PV}_source.zip"
+
+KEYWORDS="-* ~amd64 ~x86"
+LICENSE="Kryoflux-MAME"
+SLOT="0"
+
+DEPEND="app-arch/unzip"
+
+S="${WORKDIR}/capsimg_source_linux_macosx/CAPSImg"
+
+DOCS=( "${WORKDIR}/DONATIONS.txt" "${WORKDIR}/HISTORY.txt" "${WORKDIR}/RELEASE.txt" )
+
+PATCHES=( "${FILESDIR}"/add_symlink.patch )
+
+src_unpack() {
+	unpack ${A}
+
+	# Unpacked ZIP-file contains two ZIP files
+	# Use the one for Linux
+	unpack_zip capsimg_source_linux_macosx.zip
```

Comment:
Is there a specific reason you're using unpacker here? Why not just built-in `unpack`?

Reply:
No reason at all. I wasn't aware, that this works with the build-in. Removed it now.